### DISCUSSION TITLE
Custom training?

### DISCUSSION THREAD

**Discussion by enn-nafnlaus** [riffusion/riffusion#21] [Q&A]
Are there any custom tools (or workflows for existing tools) for extending the training on the model?  Would be interesting to see what quality could be achieved if the models were more specialized!  :)

**Comment by jprobichaud** [comment]
I would second that too!  I'd like to see what else we can get by fine-tuning the model some more.  Can you share the recipe you've taken a bit more?

**Comment by wedgeewoo** [comment]
Ive been messing around with using riffusion-manipulator and creating spectrograms of my own music. I then take the images generated and train an embedding or hypernetwork.

**Reply by whodaniel** [reply]
Would really love to see a video of what you're doing.
